starter_code/src/movies.js

```diff
@@ -1,20 +1,53 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
-
+  function turnHoursToMinutes (array) {
+    var moviesModified = array.map(function(e, i) {
+        var split = e.duration.split(" ");
+        var timeInMinutes = 0;
+        if (split.length === 1) {
+          if (split[0].includes("h")){
+              timeInMinutes += parseInt(split[0].replace("h", "")) * 60;
+            } else if (split[0].includes("min")) {
+              timeInMinutes += parseInt(split[0].replace("min", ""))
+            }
+        } else {
+            timeInMinutes += parseInt(split[0].replace("h", "")) * 60;
+            timeInMinutes += parseInt(split[1].replace("min", ""));
+            timeInMinutes += e.duration[3] = " mins";
+        }
+        
+        return Object.assign({}, e, {duration: timeInMinutes} );
+    });
+     return moviesModified;
+} 
+
+console.log(turnHoursToMinutes(movies));
 
 // Get the average of all rates with 2 decimals 
-
+function ratesAverage(array) {
+  var moviesSum = movies.reduce(function(accumulator, moviesArray) {
```

Comment:
Habéis puesto `var moviesSum = movies.reduce(function(accumulator, moviesArray)` cuando movies no es nada, vosotros le estáis pasando un array, no las movies. Por tanto donde estáis escribiendo movies tenéis que escribir array. `var moviesSum = array.reduce(function(accumulator, moviesArray)`
Una vez resuelto este problema tendréis que solucionar un caso porque un rate que se os pasa es tipo string lo que os contamina todo, por tanto la solución que siempre transformar el valor entrante en number. Quedaría una cosa así.
```
function ratesAverage(array) {
  var moviesSum = array.reduce(function(accumulator, moviesArray) {
    return accumulator + parseInt(moviesArray.rate);
  }, 0);
  return moviesSum.toFixed(2) / array.length;;
}
```